interface.go

```diff
@@ -85,7 +85,7 @@ type GlobalLookupFactory interface {
 	// Help text for the CLI
 	Help() string
 	// Return a single scanner which will scan a single host
-	MakeRoutineFactory() (RoutineLookupFactory, error)
+	MakeRoutineFactory(int) (RoutineLookupFactory, error)
```

Comment:
My point was to just try to hit some of the low hanging fruit, such as comment style because they're easy, not to imply you had to fix all the go lint errors.